nms/packages/magmalte/fbc_js_core/ui/host/OrganizationDialog.js

```diff
@@ -122,42 +173,53 @@ export default function (props: Props) {
     allNetworks,
     shouldEnableAllNetworks,
     setShouldEnableAllNetworks,
+    edit: props.edit,
+    hideAdvancedFields: props.hideAdvancedFields,
   };
   const onSave = async () => {
     if (currentTab === 0) {
       if (!organization.name) {
         setCreateError('Name cannot be empty');
         return;
       }
-      const payload = {
+      const newOrg = {
         name: organization.name,
         networkIDs: shouldEnableAllNetworks
           ? allNetworks
-          : Array.from(organization.networkIDs || []),
+          : Array.from(organization.networkIDs || []).sort(),
         customDomains: [], // TODO
-        // tabs: Array.from(tabs),
+        // default tab is nms
+        tabs: Array.from(organization.tabs || ['nms']),
```

Comment:
Could you add a TODO here - `tabs` shouldn't be an option, as it's always the same. This is a holdover from when this code was shared between different FBC projects.